src/index.ts

```diff
@@ -137,6 +145,37 @@ export class WaxJS {
     return this.user.account;
   }
 
+  private async getChainInfoByChainName(chainName: string): Promise<any> {
+    const response = await this.registryRpc.get_table_rows({
+      json: true,
+      code: "registry.wax",
+      scope: "registry.wax",
+      table: "chains",
+    });
+    const chainInfo: any | null = response.rows.find(row => row.chain_name === chainName);
+
+    if (!chainInfo) {
+      throw new Error('Chain name not found.');
+    }
+
+    return chainInfo;
+  }
+
+  public async setupChain(chainName: string | null): Promise<void> {
+    if(!chainName) {
```

Comment:
let's remove this check and entire if block

Reply:
updated 👍